lib/rentlinx/validators/phone_validator.rb

```diff
@@ -0,0 +1,14 @@
+module Rentlinx
+  class PhoneValidator < BaseValidator
+    def validate
```

Comment:
Should these be private methods? If not, should the result of `validate` be cached in the event the validation is expensive?